build_sysext

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Copyright (c) 2023 Flatcar Linux. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file. Script to generate sysext. See systemd-sysext(8).
+# Prerequisite that you've run build_packages and build_image
+
+SCRIPT_ROOT=$(dirname "$(readlink -f "$0")")
+. "${SCRIPT_ROOT}/common.sh" || exit 1
+
+# Script must run inside the chroot
+assert_inside_chroot
+assert_root_user
+
+# All these are used to set up the 'BUILD_DIR' variable
+DEFINE_string board "${DEFAULT_BOARD}" \
+  "The board to build a sysext for."
+DEFINE_string output_root "${DEFAULT_BUILD_ROOT}/images" \
+  "Directory in which to place sysext images and intermediate files"
+DEFINE_string group "developer" \
+  "The normal image build group"
+DEFINE_integer build_attempt 1 \
+  "The build attempt for this build."
+
+FLAGS_HELP="USAGE: build_sysext [flags] [sysext name] [packages to build].
+This script is used to build a Flatcar sysext. 
+
+Examples:
+
+Build a systemd-sysext image named command-extras with package_1, package_2 and package_3 for amd64:
+sudo build_sysext --board=amd64-usr command-extras package_1 package_2 package_3
+
+Builds a systemd-sysext image named command-extras with package_1, package_2 and package_3 for arm64:
+sudo build_sysext --board=arm64-usr command-extras package_1 package_2 package_3
+
+"
+
+show_help_if_requested "$@"
+
+# Parse command line
+FLAGS "$@" || exit 1
+
+eval set -- "${FLAGS_ARGV}"
+
+. "${BUILD_LIBRARY_DIR}/release_util.sh" || exit 1
+
+. "${BUILD_LIBRARY_DIR}/toolchain_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/board_options.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/build_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/dev_container_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/test_image_content.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/torcx_manifest.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/vm_image_util.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/cros_vm_constants.sh" || exit 1
+. "${BUILD_LIBRARY_DIR}/oem_sysext_util.sh" || exit 1
+
+set -euo pipefail
+
+cleanup() {
+  umount "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" 2>/dev/null || true
+  rm -rf "${BUILD_DIR}/fs-root" "${BUILD_DIR}/install-root" "${BUILD_DIR}/workdir" || true
+}
+
+export SYSEXTNAME="${1-}"
+
+if [ -z "$SYSEXTNAME" ]; then
+    echo "Error: No sysext name passed"
+    show_help_if_requested -h
+fi
+
+# Set up trap to execute cleanup() on script exit
+trap cleanup EXIT
+
+ARCH=$(_get_sysext_arch "${FLAGS_board}") 
+
+source <(grep -P '^(FLATCAR_VERSION|FLATCAR_VERSION_ID)=' /etc/os-release)
```

Comment:
These variables (`FLATCAR_VERSION` and `FLATCAR_VERSION_ID` are set in `common.sh` so they could be used directly.

or let's use `${REPO_MANIFESTS_DIR}` instead of `${BASH_SOURCE[0]}`